### ISSUE TITLE
Category - Narrow spacing between search field and "Recent" header

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#37328]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.44-0
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

## Action Performed:
1. Go to staging.new.expensify.com
2. Go to workspace chat
3. Open manual request page
4. Enter amount > Next
5. Click Show more > Category

## Expected Result:
There will be adequate spacing between the search field and "Recent" header

## Actual Result:
The spacing between search field and "Recent" is narrow

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

![Bug6394362_1709055607590!Screenshot_2024-02-27_at_13 50 42](https://github.com/Expensify/App/assets/78819774/c224bc9b-2f75-4350-944d-dbf89dbff071)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by lanitochka17** [comment]
We think that this bug might be related to #wave5
CC @dylanexpensify

**Comment by godofoutcasts94** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Category - Narrow spacing between search field and "Recent" header
### What is the root cause of that problem?
Over here, there is no spacing
```
<Text style={[styles.ph5, styles.pv3]}>{translate('iou.categorySelection')}</Text>
            <CategoryPicker
                selectedCategory={transactionCategory}
                policyID={report.policyID}
                onSubmit={updateCategory}
            />
```
### What changes do you think we should make in order to solve the problem?

We need to add spacing here 
### What alternative solutions did you explore? (Optional)
N/A

**Comment by ShridharGoel** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Spacing between header and content is narrow.

### What is the root cause of that problem?
There's no padding in View of `renderSectionHeader`. 

### What changes do you think we should make in order to solve the problem?
Add padding for the header in BaseSelectionList
```
const renderSectionHeader = ({ section }: { section: SectionListDataType<TItem> }) => {
        if (!section.title || isEmptyObject(section.data)) {
            return null;
        }

        return (
            <View style={[styles.optionsListSectionHeader, styles.justifyContentCenter, { paddingTop: 10 }]}>
                {/* 10 is just an example */}
                <Text style={[styles.ph4, styles.textLabelSupporting]}>{section.title}</Text>
            </View>
        );
    };
```

**Comment by godofoutcasts94** [comment]
Updated [Proposal](https://github.com/Expensify/App/issues/37328#issuecomment-1967295147)

**Comment by puneetlath** [comment]
Definitely a bug, but doesn't need to block the deploy. Removing the label.

**Comment by dragnoir** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.

List header needs more spacing.

### What is the root cause of that problem?

The value of `marginTop` for `optionsListSectionHeader` is 8.

### What changes do you think we should make in order to solve the problem?

Change the value on `optionsListSectionHeader` style for `marginTop` from 8 to 20.
```diff
optionsListSectionHeader: {
-   marginTop:  8,
+   marginTop:  20,
	marginBottom:  4,
},
```

### What alternative solutions did you explore? (Optional)

**Comment by abdulrahuman5196** [comment]
Hi, Will take a look in my morning.

**Comment by abdulrahuman5196** [comment]
Reviewing now

**Comment by abdulrahuman5196** [comment]
This is straightforward styling issue with multiple possible ways to update the styling. So going ahead with the first proposal since all proposals having similar logic at different places.

@godofoutcasts94 's proposal here https://github.com/Expensify/App/issues/37328#issuecomment-1967295147 looks good and works well.

🎀 👀 🎀
C+ Reviewed

**Comment by godofoutcasts94** [comment]
> This is straightforward styling issue with multiple possible ways to update the styling. So going ahead with the first proposal since all proposals having similar logic at different places.
> 
> @godofoutcasts94 's proposal here [#37328 (comment)](https://github.com/Expensify/App/issues/37328#issuecomment-1967295147) looks good and works well.
> 
> 🎀 👀 🎀 C+ Reviewed

Thanks for assigning it to me @abdulrahuman5196 ! Tomorrow PR will be ready for review for sure because actually I am out somewhere that's why

**Comment by aldo-expensify** [comment]
Adding @Expensify/design to confirm we want this change and to make it right if we do

**Comment by dannymcclain** [comment]
Would this increase the spacing below _every_ search in the app or just for categories?

I compared the spacing for a new chat, money request, and category, and they all look the same to me (these screenshots are from staging).

![image](https://github.com/Expensify/App/assets/7516624/34eb77b2-96e1-499a-90dc-ae7230b41eaf)

cc @Expensify/design (on purpose this time 😁) I feel like this might be a do nothing situation.

**Comment by shawnborton** [comment]
Yeah, I agree with you! I think we can just close this. 

@lanitochka17  can you help us understand why you decided that this was a bug? Mostly want to understand if there was a test that failed or something else.

**Comment by godofoutcasts94** [comment]
Hey @shawnborton and @dannymcclain. Is there anything that i would get if the issue is closed ? because I tried my level best and also put on some work and my proposal got selected as well.

**Comment by shawnborton** [comment]
If we close this out (which I anticipate that we will), then there would be no payments awarded since we deem this to not be a bug.

**Comment by shawnborton** [comment]
@lanitochka17 @puneetlath @aldo-expensify please feel encouraged to tag the design team earlier on in the triage process so we can make these determinations faster in the future - thanks!

**Comment by godofoutcasts94** [comment]
No problem thanks for updating @shawnborton

**Comment by shawnborton** [comment]
Of course! We definitely appreciate your proposal though and hope to see you around on other issues in the repo 😄

**Comment by godofoutcasts94** [comment]
Yes @dukenv0307 Thanks! And yes yesterday I was also not in front of my system. So I didn't see that but yes that's true what @dukenv0307 is stating that previously we were using `OptionSelector` in here - `src/components/CategoryPicker.tsx`, but now after this [PR](https://github.com/Expensify/App/pull/35567). We have added `SelectorList`. The `margin-top` used to be `20px` but now its just `8px`. 

So, please have a look at it @shawnborton

**Comment by godofoutcasts94** [comment]
I am pretty confused. What should I do? Should I raise a PR for review or not ? @abdulrahuman5196

**Comment by abdulrahuman5196** [comment]
> I am pretty confused. What should I do? Should I raise a PR for review or not ? @abdulrahuman5196

@godofoutcasts94 PR shouldn't be raised until you have been assigned(If assigned after discussions)


>I think the bug here is the space between the search input and the section header is less than before. In the past, we used OptionSelector, section header had margin-top 20px but now it's 8px when we use SelectionList.

I am not sure comparing it to previous view would be the right path. Anyways it would be design decision to maintain the current existing spacing or have more spacing. I was under the impression this was already design team reviewed since the bug as already a deployment blocker.

**Comment by godofoutcasts94** [comment]
Okay @abdulrahuman5196 ! So I have to wait for the design team to come up with the idea or thing what they want then only we can proceed further right ?

**Comment by godofoutcasts94** [comment]
Hey @shawnborton! Can you please have a look on [this](https://github.com/Expensify/App/issues/37328#issuecomment-1978001149) comment and let me know what needs to become done as @abdulrahuman5196 initally assigned me this issue but right I can't do anything about it. So please decide on this.

**Comment by shawnborton** [comment]
I guess we could add extra space below the search input, or we just do nothing and close this out. Curious what everyone else thinks.

**Comment by aldo-expensify** [comment]
No strong opinion, I'll go with what you think is best from a design point of view :P. if both options feel equivalent, I would close.

**Comment by godofoutcasts94** [comment]
> I guess we could add extra space below the search input, or we just do nothing and close this out. Curious what everyone else thinks. 

So my proposal is doing the same @shawnborton but yes whatever you decide that will be fine.

**Comment by dannymcclain** [comment]
I think we should close.

**Comment by strepanier03** [comment]
I'm happy to close this and have everyone focus on higher value bugs. That's my vote.
